### DISCUSSION TITLE
Additional Book Metadata

### DISCUSSION THREAD

**Discussion by jpmorr** [quarto-dev/quarto-cli#2059] [Q&A] [answered]
I'm going through the available metadata for a book project and along with `title`, `subtitle`, `abstract` etc, it feels like there should be options for setting the edition, ISBN and DOI which could be used in the creation of title pages.

Is this something that be done by just adding the variables into the `yaml`? Would this make the variables available for use in the `template-partials`? I've just tried out the examples from [quarto titlepages](https://nmfs-opensci.github.io/quarto_titlepages) and I's like to include information such as edition and ISBN.

**Comment by mcanouil** [comment, marked as answer]
See #2060
